startup

```diff
@@ -1,11 +1,3 @@
 echo "==================================>install bower"
 python manage.py bower install
-
```

Comment:
@Emmanuel-Dominic any reason as to why this file was modified yet you are not using it?